### PR TITLE
OLS-160: Definition of Done

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Definition of Done

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [x] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #[OLS-160](https://issues.redhat.com//browse/OLS-160): Definition of Done

### CODE REVIEW

**Comment by tisnik** [comment]
It's there to be discussed first!

File: docs/definition_of_done.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Definition of Done
+
+## A deliverable is to be considered “done” when
+
+* Code is complete, commented, and merged to the relevant release branch
+* User facing documentation written (where relevant)
+* Acceptance criteria are verified and fulfilled
```
Comment (bparees) [inline docs/definition_of_done.md:7]:
```suggestion
* Acceptance criteria in the related Jira ticket(where applicable) are verified and fulfilled
```

**Review by bparees — COMMENTED** [review]
@tisnik can you put this in the contribution guide instead of having it as a separate file? Thanks

File: docs/definition_of_done.md (hunk 1 above)
Comment (tisnik) [reply docs/definition_of_done.md]: done

Comment (tisnik) [reply docs/definition_of_done.md]: @bparees ^

**Pull request merged**